bin/cli.js

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env node
 'use strict';
 
+
+const extensionArgs = ['--extensions', 'hbs'];
```

Comment:
```suggestion
```

Reply:
this broke things :-\

Reply:
I think because previously, the file path was the last thing, and now the extension is. The path needs to be last :-\

Reply:
or, I'm pulled to thin, and don't know how this works:
`  await execa('../../../bin/cli.js', ['http://localhost:4200', 'app'], execOpts);`

Reply:
anyway, to make this PR worth merging, the integration tests should pass *

Reply:
> The path needs to be last :-\

Ya, not sure why you think that. `runTransform` is defined like:

```js
function runTransform(binRoot, transformName, args, extensions = DEFAULT_EXTENSIONS) {
```

So the paths aren't last, they are the third argument, and the extensions (comma separated string) is last.

Reply:
I'll poke at the branch though, maybe I made a mistake in the impl

Reply:
> Ya, not sure why you think that. runTransform is defined like:

legit! I did not check that out. I think my assumption was a red herring.